packages/build/tests/core/tests.js

```diff
@@ -323,6 +323,30 @@ test('Print warning when headers file is missing from publish directory', async
   })
 })
 
+test.serial('Successfully builds ES module function with feature flag', async (t) => {
+  const spy = sinon.spy(zipItAndShipIt, 'zipFunctions')
+
+  await runFixture(t, 'functions_es_modules', {
+    flags: { featureFlags: { buildbot_es_modules_esbuild: true } },
+  })
+
+  const { args: callArgs } = spy.getCall(0)
+  t.true(callArgs[2].featureFlags.defaultEsModulesToEsbuild)
+
+  zipItAndShipIt.zipFunctions.restore()
```

Comment:
any tests spying on the same thing need to be `serial` and `restore()`ed at the end (there are only 3 total, now, so it didn't feel worth it to spy `beforeEach`)

Reply:
This makes sense :+1:

Reply:
I liked your point about a test crashing, so I moved it to an `afterEach.always`